### DISCUSSION TITLE
`x-modelable` does not respect a debounced `x-model`

### DISCUSSION THREAD

**Discussion by mattgreenfield** [alpinejs/alpine#2812] [7. Bugs (archived)]
I'm using `x-modelable` within a blade component so that I can have access to the `x-model` value and sync with it. This is working correctly.

However, if I want to debounce the updates to the `x-model` value, it is not debounced.

I've created a demo here https://codesandbox.io/s/alpinejs-playground-eq1i7s?file=/index.html

You'll see that the first range input directly changes `parentval` with `x-model.debounce.900="parentval"` and that it is correctly debounced.
The second input sets `value` which in turn sets `parentval` via `x-modelable`. The value is changed instantly, not waiting for the debounce.

**Comment by mattgreenfield** [comment]
Relates to https://github.com/alpinejs/alpine/discussions/2764

**Reply by SimoTod** [reply]
I know it's not really addressing the bug but you can achieve this by debouncing the inner x-model

```html
      <div
        x-data="{ value: 10 }"
        x-model="parentval"
        x-modelable="value"
      >
        <input type="range" min="0" max="200" step="10" x-model.debounce.900="value" />
      </div>
 ```
